src/main/java/gov/cabinetofice/gapuserservice/web/LoginControllerV2.java

```diff
@@ -39,8 +39,12 @@ public class LoginControllerV2 {
     private final CustomJwtServiceImpl customJwtService;
     private final ApplicationConfigProperties configProperties;
 
+    public static final String PRIVACY_POLICY_PAGE_VIEW = "privacy-policy";
+
     public static final String NOTICE_PAGE_VIEW = "notice-page";
 
+    private static final String REDIRECT_URL_COOKIE = "redirectUrl";
```

Comment:
I think this should come from an env variable so we don't need to define / modify it more than once

Reply:
This is used to configure @Cookie annotations and must be available at compile time, so we can't use an env variable for this

Reply:
Had another review - it's likely only going to get here anyway I think